new/components/vcloud/update-vcloud-vm.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
+uname -a
+
+# update system
+yum update --skip-broken -y
```

Comment:
About compilers: vcloud plugin required complier for build several modules that don't exist in ubuntu repositories,  so when we installed old manager and tried to use with vcloud we had issues with it. 

This blueprint have not fully tested with vcloud plugin deployments, so i will ask separately about merge later after full test and maybe we will drop requirements to install compilers/update system.